.github/workflows/deploy.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish package to the Maven Central Repository
+on:
+  release:
+    types: [created]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: '22' # or 11?
+          distribution: 'temurin'
+          server-id: ossrh
+          server-username: ${{secrets.OSSRH_USERNAME}}
+          server-password: ${{secrets.OSSRH_TOKEN}}
+
+      - name: Download blosc jar
+        run: |
+          mkdir -p ../blosc-java/target
+          curl https://static.webknossos.org/misc/blosc-java-0.1-1.21.4-SNAPSHOT.jar -o ../blosc-java/target/blosc-java-0.1-1.21.4-SNAPSHOT.jar
```

Comment:
I assume as a next step this dependency is probably the next one to bring into the zarr-developers organization and deploy to Maven Central?

Reply:
`blosc-java` is actually already on Maven https://mvnrepository.com/artifact/com.scalableminds/blosc-java

Reply:
Great. So is this download step necessary or can the build simply use its standard dependency mechanism?

Reply:
@brokkoli71 can you check if we can use the maven dep directly?

Reply:
It works and I've changed it accordingly  👍🏼